content/release-notes/2022-09-02.md

```diff
@@ -4,10 +4,10 @@ label: 'Console'
 
 ### What's new
 
-- UI: Added support for passing the OAuth app name to the client.
+- UI: Added support for passing an OAuth app name to a client.
```

Comment:
'added support for passing' sownds awkward, but i don't have an immediate suggestion on how to imrove this :(

Reply:
Changed to "Added functionality that enables providing an OAuth app name to a client."